api/v1alpha4/openstackmachine_types.go

```diff
@@ -19,6 +19,7 @@ package v1alpha4
 import (
```

Comment:
Now that v1beta1 is created, update api/v1beta1/openstackmachine_types.go instead of this, please.